### PR TITLE
Response Entitlements

### PR DESCRIPTION
**Pull request by blainsmith**

### CODE REVIEW

File: lib/middleware/response-entitlements.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	return function responseEntitlements(req, res, next) {
+		const channel = req.identity.channel;
+		const viewer = req.identity.viewer;
+
+		// Default the evaluator to return true
+		let entitledEvaluator = yes;
+
+		// If auth is enabled then reset the evaluator to the one set on the channel
+		if (channel.features.authentication.enabled) {
+			entitledEvaluator = eval(`(${channel.features.authentication.evaluator})`); // eslint-disable-line no-eval
```
Comment (brkattk) [inline lib/middleware/response-entitlements.js]: should you wrap this in `try`, or are we expecting that this is valid?

Comment (blainsmith) [inline lib/middleware/response-entitlements.js]: Yeah. I was thinking of a nice way to handle that. We will be the ones writing these for now, but that would be a good safety net. If it fails we should probably default `entitlements = false` then.

Comment (brkattk) [inline lib/middleware/response-entitlements.js]: definitely needs to send out an error on the bus, too

**Comment by blainsmith** [comment]
@brkattk Updated PR per your suggestions.

**Pull request merged**